linux_sql/README.md

```diff
@@ -0,0 +1,43 @@
+
+# Introduction
```

Comment:
Readme file should be descriptive in terms of what the application/script does, how to get it started and how to use it.